src/snapshot/readme.md

```diff
@@ -24,12 +24,12 @@ result = snapshot({ source, clock, fn });
 ```
 
 - Call `fn` with data from `source` while `clock` triggered, and create store with the value
-- If function in `shape` returns `undefined`, the update will be skipped.
+- If `fn` returns `undefined`, the update will be skipped.
 
 ### Arguments
 
 1. `source` ([_`Store`_]) — Source store, data from this unit passed to `fn`
-2. `clock` ([_`Event`_]) — Trigger event
+2. `clock` ([_`Event`_], [_`Effect`_], [_`Store`_]) — Trigger unit
```

Comment:
Actually Unit is a domain and a Scope
https://github.com/effector/effector/blob/master/packages/effector/index.d.ts#L3009

what about to explicitly set `Event<T> | Store<T> | Effect<T, any, any>` ?